purview/models/__init__.py

```diff
@@ -19,6 +19,8 @@
 
 # Overrides from https://github.com/neo4j-contrib/neomodel/pull/327
 # These should be removed once a new version is released
+
```

Comment:
This seems unnecessary.

Reply:
It was throwing a linting error :(

Reply:
Oh alright. I think if you rebase develop on master and rebase your branch on develop this should be resolved.